vendor/firebase/shim.js

```diff
@@ -1,9 +1,18 @@
 /* globals Firebase */
 
-define('firebase', [], function() {
-  "use strict";
+(function() {
+  function fakeFirebase() {
+    var error = "Firebase is not available. Either the bower package is missing or your app is in Fastboot Mode.";
+    this.to = function() {
```

Comment:
I actually don't think it is needed. I was copying this stuff from https://github.com/runspired/ember-tweenlite/blob/master/vendor/shims/tweenlite.js